style.css

```diff
@@ -345,10 +351,7 @@ body {
   margin-top: 2rem;
   padding: 3rem;
   padding-top: 7rem;
-  border-top-left-radius: 100px 70px;
-  display: flex;
-  flex-direction: column;
-  align-content: center;
+  border-top-left-radius: 5em;
 }
 
 .contact-content h1 {
```

Comment:
- [OPTIONAL] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/). 💯